config/manager/manager.yaml

```diff
@@ -13,31 +12,31 @@ spec:
       labels:
         name: mongodb-kubernetes-operator
     spec:
-      serviceAccountName: mongodb-kubernetes-operator
       containers:
-        - name: mongodb-kubernetes-operator
-          image: quay.io/mongodb/mongodb-kubernetes-operator:0.6.0
-          command:
-            - /usr/local/bin/entrypoint
-          imagePullPolicy: Always
-          env:
-            - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
-            - name: POD_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
-            - name: OPERATOR_NAME
-              value: "mongodb-kubernetes-operator"
-            - name: AGENT_IMAGE # The MongoDB Agent the operator will deploy to manage MongoDB deployments
-              value: quay.io/mongodb/mongodb-agent:10.29.0.6830-1
-            - name: VERSION_UPGRADE_HOOK_IMAGE
-              value: quay.io/mongodb/mongodb-kubernetes-operator-version-upgrade-post-start-hook:1.0.2
-            - name: READINESS_PROBE_IMAGE
-              value: quay.io/mongodb/mongodb-kubernetes-readinessprobe:1.0.3
-            - name: MONGODB_IMAGE
-              value: "library/mongo"
-            - name: MONGODB_REPO_URL
-              value: "registry.hub.docker.com"
+      - command:
+        - /usr/local/bin/entrypoint
+        env:
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: OPERATOR_NAME
+          value: mongodb-kubernetes-operator
+        - name: AGENT_IMAGE
+          value: quay.io/mongodb/mongodb-agent:10.29.0.6830-1
+        - name: VERSION_UPGRADE_HOOK_IMAGE
+          value: quay.io/mongodb/mongodb-kubernetes-operator-version-upgrade-post-start-hook:1.0.2
+        - name: READINESS_PROBE_IMAGE
+          value: quay.io/mongodb/mongodb-kubernetes-readinessprobe:1.0.4
+        - name: MONGODB_IMAGE
+          value: library/mongo
+        - name: MONGODB_REPO_URL
+          value: registry.hub.docker.com
+        image: quay.io/mongodb/mongodb-kubernetes-operator:0.6.0
+        imagePullPolicy: Always
+        name: mongodb-kubernetes-operator
+      serviceAccountName: mongodb-kubernetes-operator
```

Comment:
the new release tool was run in order to prevent differences in formatting going forward, there are no changes in versions here.